scripts/onelink-smart-script.js

```diff
@@ -11,7 +11,7 @@ class OneLinkUrlGenerator {
                     campaignKeysList = [],
                     campaignStaticValue = null,
                     pidOverrideList = [],
-                    gclIdParam = 'af_sub5',
+                    gclIdParam = 'af_sub1',
```

Comment:
@miguelsendwave why would you like to change it?
This is the default we have chosen, as it is the most common among our customers.